app/models/concerns/findings/issues.rb

```diff
@@ -9,6 +9,46 @@ module Findings::Issues
     accepts_nested_attributes_for :issues, allow_destroy: true, reject_if: :all_blank
   end
 
+  def issues_amount
+    issues.sum &:amount
+  end
+
+  def get_amount_by_impact
+    amount = issues_amount
+
+    Hash[amount_by_impact.to_a.reverse].to_hash.detect { |id, import| amount >= import }
```

Comment:
¿Porqué el `to_hash`?

Aparte, por lo que veo podrías hacer algo como:
```suggestion
    amount_by_impact.reverse_each.to_h.detect { |id, import| amount >= import }
```
Y ahora, ¿Porqué se llama import la variable?

Reply:
Por que es el importe del impacto

Reply:
Pero import no es importe.

Reply:
Jaja, cuando tenes razón, tenes razón!!!